app/src/org/commcare/dalvik/activities/CrashWarningActivity.java

```diff
@@ -54,7 +54,7 @@ private void setupButtons() {
         closeButton.setText(Localization.get("crash.warning.button"));
         closeButton.setOnClickListener(new View.OnClickListener() {
             public void onClick(View v) {
-                restartCommCare();
+                CommCareApplication._().restartCommCare(CrashWarningActivity.this);
```

Comment:
Calls to `restartCommCare` shouldn't be delegated through an instance (like `_()`) because it is a static method.